airbyte-cdk/bulk/core/load/src/main/kotlin/io/airbyte/cdk/data/JsonSchemaToAirbyteType.kt

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright (c) 2024 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.cdk.data
+
+import com.fasterxml.jackson.databind.node.ObjectNode
+
+class JsonSchemaToAirbyteType {
+    fun convert(schema: ObjectNode): AirbyteType {
```

Comment:
Well, we're going to have to write the reverse mapper to get this back into json for test inputs anyway

So, we'll have a map <-> reverse map

So, a first step for converting a destination could be to put that into the deserialization path on the old destination & then run DATs?

Reply:
Also, re the weird implementation, my existing avro/parquet path probably misses that one as well.

Reply:
fair point (I gut thought it would be hard to plug this code into an old destination, but that feels solvable) :shrug:

I tend to err conservative on these behaviors, but that's probably overcautious, given that the s3 rollout went fine 🚛